bin/deploy-wmagent.sh

```diff
@@ -191,12 +191,6 @@ unzip -q deployment.zip
 cd deployment-$CMSWEB_TAG
 set +e 
 
-### Patching Couchdb1.6
-echo -e "\n*** Applying deployment patches (couchdb1.6, etc) ***"
-wget -nv https://github.com/dmwm/deployment/pull/162.patch -O - | patch -p 1
-chmod 600 /data/certs/service{cert,key}.pem
```

Comment:
Hmm, I thought this was affecting only central site... or is it vice versa? When I ran the test without it wasn't make couch crash. Could you check as well?